### PR TITLE
Feat/#5 Update display style

### PR DESCRIPTION
**Pull request by nviethuan**
How to test: checkout to branch and run `yarn start`.

## Ref

#5

### CODE REVIEW

**Comment by nviethuan** [comment]
How to test: checkout to branch and run `yarn start`.

## Ref
https://github.com/manh-vv/scrum-poker-online/issues/5

File: src/index.html [hunk 1]
```diff
@@ -58,32 +76,50 @@ <h2>User list</h2>
...
+        </div>
+      </div>
     </script>
 
     <script type="text/x-template" id="card-room">
       <div class="mui-panel">
-        <div>
-          {{name}}
-        </div>
+        <h1 style="color: #ca1313; text-align: center">{{ name }}</h1>
```
Comment (manh-vv) [inline src/index.html]: we should make scss bem for this

File: src/index.js [hunk 2]
```diff
@@ -22,7 +22,9 @@ function onLoad() {
   Vue.component('card-room', {
     template: '#card-room',
     data: function() {
-      return {};
+      return {
+        roomName: this.$root.curRoom ? this.$root.curRoom.name : '',
```
Comment (manh-vv) [inline src/index.js]: IMO, we don't need to change file

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.mg-t {
+  margin-top: 1rem;
```
Comment (manh-vv) [inline src/style.css]: IMO, using `rem` for margin is not good practice

Comment (nviethuan) [reply src/style.css]: What should unit we use?

Comment (manh-vv) [reply src/style.css]: you can use `px`.

File: src/index.html (hunk 1 above)
Comment (manh-vv) [reply src/index.html]: anyways, you can ignore this comment

**Pull request merged**